pkg/stanza/operator/helper/transformer.go

```diff
@@ -94,10 +96,22 @@ func (t *TransformerOperator) ProcessWith(ctx context.Context, entry *entry.Entr
 
 // HandleEntryError will handle an entry error using the on_error strategy.
 func (t *TransformerOperator) HandleEntryError(ctx context.Context, entry *entry.Entry, err error) error {
+	logFields := []zap.Field{
+		zap.Any("error", err),
+		zap.Any("action", t.OnError),
+	}
+	if entry != nil {
+		toAddFields := []zap.Field{zap.Any(attrs.LogRecordOriginal, entry.Body)}
```

Comment:
I don't agree we should add the body. The body could contain sensitive information and is not necessarily the reason for the failure. Maybe the timestamp is enough?